pulp_ansible/app/models.py

```diff
@@ -34,6 +34,36 @@ class Meta:
         )
 
 
+class CollectionQueryset(models.QuerySet):
+    """
+    A custom manager for Collection objects.
+    """
+
+    def full_text_search(self, text):
+        """
+        Full-text search of the Collection object including these fields.
+
+        * namespace
+        * name
+        * version
+
+        Args:
+            text: The text to search for
+
+        Returns:
+            A queryset annotated with the .extra() query which brings in the full-text query results
+                from PostgreSQL.
+
+        """
+        return self.extra(
```

Comment:
Django documentation discourages `extra()` method usage [1]. This kind of request can be built with standard Django ORM functionality. Please refer to [2] and [3] for an example.

[1] https://docs.djangoproject.com/en/2.2/ref/models/querysets/#extra
[2] https://github.com/ansible/galaxy/blob/devel/galaxy/api/internal/search.py#L86
[3] https://github.com/ansible/galaxy/blob/devel/galaxy/api/internal/search.py#L130